### PR TITLE
Rename package to Zeiss.PiWeb.Api.Rest

### PR DESCRIPTION
**Pull request by czthiele**

### CODE REVIEW

**Comment by MichaWresche** [comment]
I am not sure about the package versions. I would prefer that all three packages have the same version. Currently, the new packages starts with version 1.0.0 while the old one has version 6.0.0

**Comment by czdietrich** [comment]
I agree, all package versions should be aligned. I think it should be enough to use the _package version_ property of _GitVersion_ for all NuGet packages.

By the way, in my opinion we should simply use _dotnet build tools_ instead of cake for building the NuGet packages. This would reduce manual work in the cake script and we would have the same results using local Visual Studio builds.

**Comment by czthiele** [comment]
All NuGet versions are now aligned. 
I will look into simplifying the build, propably in a different PR.

**Comment by MichaWresche** [comment]
There is a dependency cycle in the **Zeiss.PiWeb.Api.Rest.Dtos** package to itself. Could you check this?

File: build.cake [hunk 1]
```diff
@@ -188,7 +188,8 @@ Task("Pack_Dtos")
         },
         Dependencies             = new [] {
             new NuSpecDependency { Id = "Newtonsoft.Json", Version = "12.0.1" },
-            new NuSpecDependency { Id = "JetBrains.Annotations", Version = "2018.2.1" }
+            new NuSpecDependency { Id = "JetBrains.Annotations", Version = "2018.2.1" },
+            new NuSpecDependency { Id = "Zeiss.PiWeb.Api.Rest.Dtos", Version = "6.0.0" }
```
Comment (MichaWresche) [inline build.cake:192]: Copy'n Paste issue: Should be Zeiss.PiWeb.Api.Definitions

**Pull request merged**